guide/moderator/reviewflag.md

```diff
@@ -0,0 +1,39 @@
+## User Handbook
+
+### Reviewing Flagged Content as a Moderator
+
+As a Moderator, no doubt at some point through your journey your users will come across content that is not appropriate, off topic, spam etc. Flarum has an incredibly simple tool to deal with this, which we will outline in much greater detail in this document.
+
+To begin with, you'll need to view the content that has been flagged. To do this, go to the **Flag** at the top right of your screen, and click on it, this will produce a list of items that have currently been flagged (in the example, we only have one). Once you're ready, click that result and you'll be taken directly to the post. See the animated example below:
+
+![Animation - Flagged posts from notification area](687474703a2f2f692e696d6775722e636f6d2f6b61395236384d2e676966.gif)
+
+
+Pretty hard to miss, right?
+
+From here you can do 1 of 2 things.
+
+ - Delete the Post
+
+Is the content truly requiring mod action? then go ahead and delete the post! just be warned that this option will only **soft-delete** the post for now (we'll show you how to permanently delete it below shortly).
+![Animation - showing post soft-deletion](687474703a2f2f692e696d6775722e636f6d2f52666c6c5351782e676966.gif)
+
+ - Dismiss the Flag
+
+Don't agree that this content needs any action? then just go ahead and **Dismiss the Flag** this will then _cancel_ the report, and act as if nothing ever happened. (literally!)
+
+### Deleting the Content
```

Comment:
I really like this write-up, but you already have a doc on deleting posts. Is this necessary? 

(Then again, I want to keep it just because it's so well done.)